src/AssetManager.php

```diff
@@ -330,19 +304,19 @@ private function convertJs(AssetBundle $bundle): AssetBundle
      *
      * {@see registerJsFile()} for more details on javascript position.
      *
-     * @return AssetBundle the registered asset bundle instance
-     * @throws InvalidConfigException
+     * @throws InvalidConfigException|RuntimeException if the asset bundle does not exist or a circular dependency
```

Comment:
Isn't it always throwing `RuntimeException`?

Reply:
Fixed.